app/views/layouts/_header.html.erb

```diff
@@ -1,6 +1,6 @@
 <header>
     <div class ="logocontainer">
-        <%= link_to "Leaderboard", root_path,id: "logo" %>
+        <%= link_to "STAR BOARD", root_path,id: "logo" %>
```

Comment:
Can we make it one word please? Like starboard.
Also, we need to have "PClub Leaderboard" somewhere, maybe as a small subheading, so that we can change the theme at the end after a few years.